pilot/pkg/controller/workloadentry/workloadentry_controller.go

```diff
@@ -243,6 +277,14 @@ func (c *Controller) unregisterWorkload(entryName string, proxy *model.Proxy, di
 	if cfg.Annotations[WorkloadControllerAnnotation] != c.instanceID {
 		return nil
 	}
+
+	conTime, _ := time.Parse(timeFormat, cfg.Annotations[ConnectedAtAnnotation])
+	// The wle has reconnected to this istiod,
```

Comment:
i'm not sure this comment is accurate - this could happen for a reconnect to any istiod.

Reply:
yes, i mean this error can happen only when reconnect to same istiod. For another istiod, the controller annotation is different.